pkgs/applications/science/biology/neuron/default.nix

```diff
@@ -1,94 +1,103 @@
 { lib, stdenv
 , fetchurl
-, pkg-config
-, automake
-, autoconf
-, libtool
-, ncurses
 , readline
-, which
-, python ? null
-, useMpi ? false
+, xorg
 , mpi
-, iv
+, cmake
+, bison
+, flex
+, git
+, perl
+, gsl
+, xcbuild
+, python3
+, usempi ? false
+, useiv ? true
+, core ? false
+, rx3d ? false
```

Comment:
Why don't we unify these flags? e.g. `usempi, useiv, usecore, userx3d` or `useMpi, useIv, useCore, useRx3D`?

Reply:
This must be done because rx3d could be a package and then things would break.